packages/xo-server-sdn-controller/src/index.js

```diff
@@ -0,0 +1,810 @@
+import assert from 'assert'
+import createLogger from '@xen-orchestra/log'
+import NodeOpenssl from 'node-openssl-cert'
+import { EventEmitter } from 'events'
+import { existsSync, readFileSync, writeFile } from 'fs'
+import { filter, find, forOwn, map } from 'lodash'
+import { fromCallback, fromEvent } from 'promise-toolbox'
+
+import { OvsdbClient } from './ovsdb-client'
+
+const log = createLogger('xo:xo-server:sdn-controller')
+
+const PROTOCOL = 'pssl'
+
+const CA_CERT = '/ca-cert.pem'
+const CLIENT_KEY = '/client-key.pem'
+const CLIENT_CERT = '/client-cert.pem'
+
+const SDN_CONTROLLER_CERT = 'sdn-controller-ca.pem'
+
+const NB_DAYS = 9999
+
+exports.configurationSchema = {
```

Comment:
Either use CommonJS (`require`, `exports`, `module.exports`) or ES modules (`import`, `export`), not both.

`export configurationSchema`.